playground/optimize-deps/__tests__/optimize-deps.spec.ts

```diff
@@ -200,3 +201,16 @@ test.runIf(isServe)('error on builtin modules usage', () => {
     ]),
   )
 })
+
+test('pre bundle css require', async () => {
+  if (isServe) {
+    const response = page.waitForResponse(/@vitejs_test-dep-css-require\.js/)
+    await page.goto(viteTestUrl)
+    const content = await (await response).text()
```

Comment:
Maybe we can use this technique

https://github.com/vitejs/vite/blob/a8c38692dbdd499830c77ba45925c8990d39a9a4/playground/js-sourcemap/__tests__/js-sourcemap.spec.ts#L13

to fetch it. I think it would be a nice bonus too if we can fetch the `index.html` instead since fetching by the JS URL is a bit of an implementation detail.

Reply:
I am a little confused about how to test the js script content via fetching the `index.html`. Can you go into more detail?

Reply:
Hmm good point. Actually I'm fine with this technique then. I didn't notice a regex is passed to `waitForResponse` 👍